code/modules/client/preferences.dm

```diff
@@ -28,6 +28,7 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 	var/toggles_chat = TOGGLES_CHAT_DEFAULT
 	var/toggles_sound = TOGGLES_SOUND_DEFAULT
 	var/toggles_gameplay = TOGGLES_GAMEPLAY_DEFAULT
+	var/middleshift_button=TRUE
```

Comment:
This should not be its own var. it should be part of `toggles_gameplay`

Reply:
Done